proto/raft_serverpb.proto

```diff
@@ -45,6 +45,8 @@ message SnapshotMeta {
     repeated SnapshotCFFile cf_files = 1;
     // true means this snapshot is triggered for load balance
     bool for_balance = 2;
+    // true means this is an empty snapshot for witness 
```

Comment:
Is this still necessary after using file less snapshot?

Reply:
yes, need it to decide how to handle the snapshot